src/Parlot/Character.cs

```diff
@@ -73,7 +73,7 @@ public static char ScanHexEscape(string text, int index, out int length)
         public static TextSpan DecodeString(TextSpan span)
         {
             // Nothing to do if the string doesn't have any escape char
-            if (span.Buffer.IndexOf('\\', span.Offset, span.Length) == -1)
+            if (span.Buffer.AsSpan(span.Offset, span.Length).IndexOf('\\') == -1)
```

Comment:
This faster? Haven't studied

Reply:
Yes, what gives most in this PR. I believe because of vectorization inside ROS methods.